pyvcloud/vcd/org.py

```diff
@@ -546,3 +554,25 @@ def get_roles_query(self, name_filter=None):
             equality_filter=name_filter,
             qfilter=org_filter)
         return query, resource_type
+
+    def change_catalog_owner(self, catalog_name, user_name):
+        """
+        Change the ownership of Catalog to a given user
+        :param catalog_name: Catalog whose ownership needs to be changed
+        :param user_name: New Owner of the Catalog
+        :return: None
+        """ # NOQA
+        if self.resource is None:
+            self.resource = self.client.get_resource(self.href)
+        catalog_resource = self.get_catalog_resource(catalog_name, True)
```

Comment:
Sending is_admin_operation=True all the time without actually checking if the current logged in user has the rights to run the command or not is a bit iffy. It would be nice if we have a check upfront that restrict usage of these commands to only admins.

Reply:
I agree, I did think of this. Currently there is no method we can check if logged-in-user is an org admin. Current org.is_admin determines if it is sysadmin or not. We can implement is_org_admin once we start working on roles. I will add a task to refactor this code and few other pieces of code where I see this gap. Let me know if you have any other ideas.